examples/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.11)
```

Comment:
I'm just curious, why require 3.11 specifically?
It may be useful to comment what CMake features are used that the requirement is pinned to this particular CMake version.

I think it is a very well-written CMake configuration.

Reply:
`include_guard()` in `ImGuiModule.cmake` file :-).
Now all targets is pure INTERFACE with INTERFACE_SOURCES. In next iteration I plan to use it in prebuild libraries without sources.
Support old versions requie more code base. We can add it later if needed.

Reply:
From a reader POV, I just meant a word of explanation would be helpful (and useful to avoid accidental lowering of version too). I mean something like here

https://gitlab.kitware.com/vtk/vtk-m/blob/9861cdecb1d4592b758c053dffea60f8293bfb4a/CMakeLists.txt#L21-27

(Disclaimer: I'm an external reader driven by CMake curiosity, not user of or contributor to ImGui)

Reply:
I think that it's not a strong enough reason to use such a modern CMake version. I think we should better aim at 3.7. 

How much benefit does `include_guard()` really add? Can't we do something similar with keeping some internal variables around?

Take a look [here](https://repology.org/metapackage/cmake/versions). Seems like going with CMake 3.7 is a safe bet for not requiring users to install/compile CMake on their system.